src/pudl_rmi/create_override_spreadsheets.py

```diff
@@ -235,7 +236,7 @@ def _prep_deprish(deprish, pudl_out) -> pd.DataFrame:
         ["utility_id_pudl", "utility_id_eia"]
     ].drop_duplicates()
     deprish.loc[:, "report_year"] = deprish.report_date.dt.year.astype("Int64")
-    deprish = deprish.merge(util_df, on=["utility_id_pudl"], how="left", validate="1:m")
+    deprish = deprish.merge(util_df, on=["utility_id_pudl"], how="left")
```

Comment:
This is a good question. I removed the `validate "1:m` because it was failing, presumably because it was just 1:1 in the case of the deprish table. I think it's because we're merging on `utility_id_pudl` which may have multipe `utility_id_eia` values encompassed within it, but we're not mapping `utility_id_pudl` to `utility_id_eia`, we're mapping `utility_id_pudl` to `utility_id_pudl` so it makes sense that it's 1:1.